.changes/next-release/Bug Fix-8f3202fe-3dbc-47dd-bb57-3a384de226ad.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "introduce time threshold for intellisense type trigger to avoid chain reaction trigger"
```

Comment:
I thought we were gonna remove this or maybe not?

Reply:
yea let me revert this change

Reply:
updated